### PR TITLE
Misspelling fixes

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
I kept in side note, some other items that could be fixed:
- responsability
- resposibility
- uninterruptable
- RC2 as implemented frm a posting from
- algorihm => as it related to an error code define, should it be fixed ?
- cought
- * constant-timeness

### CODE REVIEW

File: crypto/asn1/asn1_par.c [hunk 1]
```diff
@@ -229,7 +229,7 @@ static int asn1_parse2(BIO *bp, const unsigned char **pp, long length,
                 }
                 BIO_printf(bp, ":%u", p[0]);
             } else if (tag == V_ASN1_BMPSTRING) {
-                /* do the BMP thang */
+                /* do the BMP thing */
```
Comment (richsalz) [inline crypto/asn1/asn1_par.c]: well this was kind of a joke, but okay :)

File: crypto/bio/b_sock.c [hunk 2]
```diff
@@ -449,7 +449,7 @@ int BIO_get_accept_socket(char *host, int bind_mode)
         /*
          * '::port' enforces IPv6 wildcard listener. Some OSes, e.g. Solaris,
          * default to IPv6 without any hint. Also note that commonly IPv6
-         * wildchard socket can service IPv4 connections just as well...
+         * wildchar socket can service IPv4 connections just as well...
```
Comment (richsalz) [inline crypto/bio/b_sock.c]: should actually be "wildcard"

File: crypto/bio/bss_file.c [hunk 3]
```diff
@@ -198,7 +198,7 @@ BIO *BIO_new_fp(FILE *stream, int close_flag)
         return (NULL);
 
     BIO_set_flags(ret, BIO_FLAGS_UPLINK); /* redundant, left for
```
Comment (richsalz) [inline crypto/bio/bss_file.c]: If you're changing tihs, make it two lines with the comment a single line before the ```BIO_set_flags``` call.

File: crypto/comp/c_zlib.c [hunk 4]
```diff
@@ -86,7 +86,7 @@ static int zlib_stateful_expand_block(COMP_CTX *ctx, unsigned char *out,
                                       unsigned int olen, unsigned char *in,
                                       unsigned int ilen);
 
-/* memory allocations functions for zlib intialization */
+/* memory allocations functions for zlib initialization */
```
Comment (richsalz) [inline crypto/comp/c_zlib.c]: see initialise vs initialize.  :)  don't bother changing all of them to one or the other.

File: crypto/dh/dh_asn1.c [hunk 5]
```diff
@@ -125,7 +125,7 @@ int i2d_int_dhx(const int_dhx942_dh *a, unsigned char **pp);
 
 IMPLEMENT_ASN1_ENCODE_FUNCTIONS_const_fname(int_dhx942_dh, DHxparams, int_dhx)
 
-/* Application leve function: read in X9.42 DH parameters into DH structure */
+/* Application !leve! function: read in X9.42 DH parameters into DH structure */
```
Comment (richsalz) [inline crypto/dh/dh_asn1.c]: should be application-level function  or replace with the word "public"

File: crypto/dh/dh_lib.c [hunk 6]
```diff
@@ -6,7 +6,7 @@
  * The implementation was written so as to conform with Netscapes SSL.
  *
  * This library is free for commercial and non-commercial use as long as
- * the following conditions are aheared to.  The following conditions
+ * the following conditions are ?aheared? to.  The following conditions
```
Comment (richsalz) [inline crypto/dh/dh_lib.c]: should be adhered

File: crypto/dh/dh_lib.c [hunk 7]
```diff
@@ -31,7 +31,7 @@
  *    must display the following acknowledgement:
  *    "This product includes cryptographic software written by
  *     Eric Young (eay@cryptsoft.com)"
- *    The word 'cryptographic' can be left out if the rouines from the library
+ *    The word 'cryptographic' can be left out if the routines from the library
```
Comment (richsalz) [inline crypto/dh/dh_lib.c]: this is silly and stupid but legal stuff often is.  do NOT change a single letter in the copyright.  i will not repeat this comment everywhere. :(

**Comment by richsalz** [comment]
I made a few comments.  Please fix those as a separate commit (so that I can review just those changes) and then I'm ready to plus-one this.  Thank you!

File: crypto/dh/dh_lib.c (hunk 6 above)
Comment (mattcaswell) [inline crypto/dh/dh_lib.c]: This is the copyright text and CANNOT be changed anyway, as per Rich's comment below.

Comment (richsalz) [inline crypto/dh/dh_lib.c]: @mattcaswell is right; i didn't realize this was copyright text, ignore this change request from me.

**Comment by richsalz** [comment]
+1
Great work, thanks.  If someone else on the dev team will plus-one this, I'll pull it into master.

**Comment by t-j-h** [comment]
+1 good to add @richsalz

**Comment by mattcaswell** [comment]
Added second-review label since Tim has +1'd

**Comment by richsalz** [comment]
pushed thanks: 0d4fb84

**Pull request closed without merging**